zadaci/sa-kolokvijuma/2020/RA/T34/G7/index.rst

```diff
@@ -0,0 +1,17 @@
+.. index::
+   Datoteke ; Tekstualne
+   Statičke strukture ; Niz
+
+
+Zadatak "Kulinarsko takmicenje"
```

Comment:
Please change it to "Kulinarsko takmi**č**enje"

Reply:
Updated.